libraries/Microsoft.Bot.Builder.AI.LUIS/LuisV3PredictionOptions.cs

```diff
@@ -0,0 +1,124 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using Microsoft.Azure.CognitiveServices.Language.LUIS.Runtime.Models;
+using Newtonsoft.Json;
+
+namespace Microsoft.Bot.Builder.AI.Luis
+{
+    /// <summary>
+    /// Optional parameters for a LUIS prediction request.
+    /// </summary>
+    public class LuisV3PredictionOptions
+    {
+        /// <summary>
+        /// Gets or sets whether all intents come back or only the top one.
+        /// </summary>
+        /// <value>
+        /// True for returning all intents.
+        /// </value>
+        public bool? IncludeAllIntents { get; set; }
+
```

Comment:
Why are thee bool? 

Based on the name, I would expect only true/false, not a tri-state. Can we just have it be a bool and default to a false?

Reply:
I've changed all of these to be simple values utilizing the recognizer default unless overridden per query.  I also remove DatetimeReference and split out timeout/Telemetry in RecognizerOptions.